### ISSUE TITLE
Crash when morphing

### ISSUE THREAD

**Issue by Metalcape** [iChun/Morph#815]
Minecraft randomly crashes when I morph into something, it happens very often but not always. The game shows the "Shutting down internal server" screen for a second before crashing. I think it might be due to some other mods I'm using. Here's the crash report:
http://openeye.openmods.info/crashes/6c7e5a48288210be5fd426b5ae0b74e0

**Comment by Tamaized** [comment]
Are you using the latest Aether? Based on the file name it doesn't look like the latest to me, can't really tell though since that log doesn't show everything :/

**Comment by Metalcape** [comment]
Now that you mention it, I think it's version 1.2, I'll try with the latest version. If you need the full crash report here it is:
```
[17:06:06] [Client thread/INFO] [STDOUT]: [net.minecraft.client.Minecraft:func_71377_b:349]: ---- Minecraft Crash Report ----
// Hey, that tickles! Hehehe!

Time: 27/05/16 17.06
Description: Unexpected error

java.lang.NullPointerException: Unexpected error
	at net.aetherteam.aether.player.PlayerAether.onUpdate(PlayerAether.java:260)
	at com.gildedgames.util.player.common.PlayerEventHandler.onLivingUpdate(PlayerEventHandler.java:70)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler_196_PlayerEventHandler_onLivingUpdate_LivingUpdateEvent.invoke(.dynamic)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:54)
	at cpw.mods.fml.common.eventhandler.EventBus.post(EventBus.java:140)
	at net.minecraftforge.common.ForgeHooks.onLivingUpdate(ForgeHooks.java:287)
	at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.java:1560)
	at net.minecraft.entity.player.EntityPlayer.func_70071_h_(Unknown Source)
	at net.minecraft.client.entity.EntityOtherPlayerMP.func_70071_h_(Unknown Source)
	at morph.client.core.TickHandlerClient.worldTick(TickHandlerClient.java:851)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler_284_TickHandlerClient_worldTick_ClientTickEvent.invoke(.dynamic)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:54)
	at cpw.mods.fml.common.eventhandler.EventBus.post(EventBus.java:140)
	at cpw.mods.fml.common.FMLCommonHandler.onPostClientTick(FMLCommonHandler.java:330)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:2064)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:973)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:898)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at net.aetherteam.aether.player.PlayerAether.onUpdate(PlayerAether.java:260)
	at com.gildedgames.util.player.common.PlayerEventHandler.onLivingUpdate(PlayerEventHandler.java:70)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler_196_PlayerEventHandler_onLivingUpdate_LivingUpdateEvent.invoke(.dynamic)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:54)
	at cpw.mods.fml.common.eventhandler.EventBus.post(EventBus.java:140)
	at net.minecraftforge.common.ForgeHooks.onLivingUpdate(ForgeHooks.java:287)
	at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.java:1560)
	at net.minecraft.entity.player.EntityPlayer.func_70071_h_(Unknown Source)
	at net.minecraft.client.entity.EntityOtherPlayerMP.func_70071_h_(Unknown Source)
	at morph.client.core.TickHandlerClient.worldTick(TickHandlerClient.java:851)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler_284_TickHandlerClient_worldTick_ClientTickEvent.invoke(.dynamic)
	at cpw.mods.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:54)
	at cpw.mods.fml.common.eventhandler.EventBus.post(EventBus.java:140)
	at cpw.mods.fml.common.FMLCommonHandler.onPostClientTick(FMLCommonHandler.java:330)

-- Affected level --
Details:
	Level name: MpServer
	All players: 1 total; [EntityClientPlayerMP['Metalcape'/9, l='MpServer', x=-66,54, y=75,62, z=-457,13]]
	Chunk stats: MultiplayerChunkCache: 1363, 1372
	Level seed: 0
	Level generator: ID 04 - BIOMESOP, ver 0. Features enabled: false
	Level generator options: 
	Level spawn location: World: (-64,64,-135), Chunk: (at 0,4,9 in -4,-9; contains blocks -64,0,-144 to -49,255,-129), Region: (-1,-1; contains chunks -32,-32 to -1,-1, blocks -512,0,-512 to -1,255,-1)
	Level time: 15158 game time, 30552 day time
	Level dimension: 0
	Level storage version: 0x00000 - Unknown?
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: false
	Forced entities: 88 total; [EntityZombie['Zombie'/10, l='MpServer', x=-74,53, y=47,00, z=-440,03], EntityClientPlayerMP['Metalcape'/9, l='MpServer', x=-66,54, y=75,62, z=-457,13], EntityItem['item.item.rottenFlesh'/1815, l='MpServer', x=-115,75, y=70,13, z=-496,28], EntityLostMiner['Lost Miner'/24, l='MpServer', x=-145,09, y=44,00, z=-381,88], EntitySkeleton['Skeleton'/25, l='MpServer', x=-136,91, y=59,00, z=-534,22], EntityBat['Bat'/793, l='MpServer', x=-1,41, y=16,00, z=-507,41], EntityBat['Bat'/794, l='MpServer', x=0,53, y=16,00, z=-507,75], EntityBat['Bat'/795, l='MpServer', x=0,84, y=16,00, z=-507,44], EntityFish['Fish'/28, l='MpServer', x=-134,56, y=62,34, z=-485,50], EntityBat['Bat'/796, l='MpServer', x=-1,50, y=16,00, z=-507,72], EntityFish['Fish'/29, l='MpServer', x=-130,78, y=61,78, z=-486,75], EntityFish['Fish'/31, l='MpServer', x=-133,47, y=62,34, z=-485,22], EntitySkeleton['Skeleton'/32, l='MpServer', x=-138,28, y=61,95, z=-479,25], EntityZombie['Zombie'/33, l='MpServer', x=-143,94, y=44,00, z=-385,63], EntityZombie['Zombie'/34, l='MpServer', x=-139,69, y=38,00, z=-394,41], EntityBat['Bat'/35, l='MpServer', x=-127,28, y=12,85, z=-499,97], EntitySkeleton['Skeleton'/37, l='MpServer', x=-121,50, y=41,00, z=-503,50], EntityCreeper['Creeper'/39, l='MpServer', x=-127,69, y=41,00, z=-504,34], EntityZombie['Zombie'/42, l='MpServer', x=-114,72, y=71,00, z=-500,16], EntityItem['item.item.rottenFlesh'/2346, l='MpServer', x=-78,82, y=63,13, z=-402,70], EntitySkeleton['Skeleton'/43, l='MpServer', x=-117,50, y=49,00, z=-474,50], EntityTreasureSlime['Treasure Slime'/45, l='MpServer', x=-122,63, y=17,23, z=-416,63], EntityBat['Bat'/46, l='MpServer', x=-108,50, y=39,10, z=-501,53], EntityZombie['Zombie'/47, l='MpServer', x=-109,94, y=38,00, z=-500,41], EntityGib['unknown'/2351, l='MpServer', x=-77,93, y=64,00, z=-400,29], EntityGib['unknown'/2352, l='MpServer', x=-77,82, y=63,00, z=-401,70], EntityGib['unknown'/2353, l='MpServer', x=-77,59, y=64,00, z=-401,09], EntityGib['unknown'/2354, l='MpServer', x=-77,54, y=64,00, z=-401,11], EntityDSummoner['Summoner'/51, l='MpServer', x=-113,44, y=70,00, z=-426,31], EntityGib['unknown'/2355, l='MpServer', x=-78,45, y=64,00, z=-400,15], EntityZombie['Zombie'/2611, l='MpServer', x=-69,50, y=49,00, z=-446,50], EntityGib['unknown'/2356, l='MpServer', x=-76,39, y=64,00, z=-400,27], EntityBat['Bat'/53, l='MpServer', x=-102,00, y=28,00, z=-402,50], EntityBat['Bat'/54, l='MpServer', x=-110,25, y=31,00, z=-409,56], EntitySkeleton['Skeleton'/2615, l='MpServer', x=-1,50, y=66,00, z=-534,50], EntityZombie['Zombie'/56, l='MpServer', x=-111,53, y=42,00, z=-382,06], EntityFish['Fish'/59, l='MpServer', x=-91,22, y=58,25, z=-528,06], EntityFish['Fish'/60, l='MpServer', x=-92,72, y=58,00, z=-532,50], EntityFish['Fish'/61, l='MpServer', x=-88,97, y=58,25, z=-530,97], EntityFish['Fish'/62, l='MpServer', x=-95,03, y=57,59, z=-526,63], EntityRocketCreeper['Rocket Creeper'/63, l='MpServer', x=-93,56, y=56,00, z=-448,28], EntitySkeleton['Skeleton'/64, l='MpServer', x=-83,63, y=57,00, z=-446,91], EntitySkeleton['Skeleton'/66, l='MpServer', x=-77,50, y=47,00, z=-441,50], EntityCreeper['Creeper'/67, l='MpServer', x=-66,50, y=15,00, z=-419,50], EntityTreasureSlime['Treasure Slime'/68, l='MpServer', x=-66,31, y=14,00, z=-416,69], EntityZombieHorse['Zombie Horse'/70, l='MpServer', x=-78,25, y=70,00, z=-393,00], EntityZombie['Zombie'/81, l='MpServer', x=-48,50, y=26,00, z=-431,50], EntityZombie['Zombie'/2641, l='MpServer', x=-26,50, y=52,00, z=-415,50], EntityHauntedTool['Haunted Tool'/82, l='MpServer', x=-53,03, y=21,00, z=-419,50], EntityWitch['Witch'/84, l='MpServer', x=-52,50, y=23,00, z=-408,50], EntityZombie['Zombie'/85, l='MpServer', x=-48,50, y=56,00, z=-415,50], EntitySkeleton['Skeleton'/86, l='MpServer', x=-57,50, y=65,00, z=-410,91], EntityCreeper['Creeper'/87, l='MpServer', x=-51,50, y=68,00, z=-403,50], EntityCreeper['Creeper'/90, l='MpServer', x=-39,41, y=55,00, z=-466,00], EntitySkeleton['Skeleton'/91, l='MpServer', x=-47,50, y=22,00, z=-433,50], EntityZombie['Zombie'/92, l='MpServer', x=-46,13, y=28,00, z=-447,47], EntityBat['Bat'/93, l='MpServer', x=-37,28, y=28,00, z=-435,25], EntityCreeper['Creeper'/96, l='MpServer', x=-36,50, y=68,00, z=-409,50], EntityZombie['Zombie'/110, l='MpServer', x=-30,00, y=27,00, z=-444,44], EntitySpider['Spider'/111, l='MpServer', x=-21,97, y=64,00, z=-440,22], EntityBat['Bat'/113, l='MpServer', x=-31,63, y=11,79, z=-390,16], EntitySkeleton['Skeleton'/115, l='MpServer', x=-25,50, y=20,00, z=-399,50], EntitySkeleton['Skeleton'/121, l='MpServer', x=-6,50, y=69,00, z=-529,50], EntityZombie['Zombie'/122, l='MpServer', x=-6,35, y=63,00, z=-516,07], EntitySquid['Squid'/126, l='MpServer', x=-16,34, y=59,34, z=-419,03], EntitySquid['Squid'/127, l='MpServer', x=-5,44, y=59,34, z=-424,84], EntitySquid['Squid'/128, l='MpServer', x=-14,34, y=57,28, z=-414,16], EntitySkeleton['Skeleton'/2177, l='MpServer', x=4,94, y=13,00, z=-414,50], EntitySkeleton['Skeleton'/2178, l='MpServer', x=8,50, y=12,00, z=-417,50], EntityItem['item.item.rottenFlesh'/1928, l='MpServer', x=-116,59, y=73,13, z=-508,38], EntityBat['Bat'/138, l='MpServer', x=8,75, y=25,00, z=-438,00], EntityDMimic['Mimic Chest'/140, l='MpServer', x=2,50, y=15,00, z=-397,50], EntitySkeleton['Skeleton'/141, l='MpServer', x=5,50, y=15,00, z=-399,50], EntitySkeleton['Skeleton'/142, l='MpServer', x=1,50, y=15,00, z=-398,50], EntitySkeleton['Skeleton'/143, l='MpServer', x=9,50, y=22,00, z=-395,50], EntityDSkeletonWarrior['Skeleton Warrior'/1681, l='MpServer', x=4,50, y=12,00, z=-417,50], EntityZombie['Zombie'/1691, l='MpServer', x=-58,50, y=21,00, z=-435,50], EntityZombie['Zombie'/1692, l='MpServer', x=-57,50, y=21,00, z=-436,50], EntityItem['item.item.rottenFlesh'/2205, l='MpServer', x=-103,53, y=70,13, z=-422,34], EntityDMimic['Mimic Chest'/1693, l='MpServer', x=-70,50, y=16,00, z=-414,50], EntityZombie['Zombie'/1694, l='MpServer', x=-2,50, y=16,00, z=-510,50], EntityCreeper['Creeper'/1695, l='MpServer', x=-1,50, y=16,00, z=-506,50], EntityCreeper['Creeper'/1696, l='MpServer', x=-0,50, y=16,00, z=-511,50], EntitySkeleton['Skeleton'/1453, l='MpServer', x=5,50, y=12,00, z=-417,50], EntitySkeleton['Skeleton'/2477, l='MpServer', x=0,73, y=16,00, z=-507,67], EntityDMimic['Mimic Chest'/2482, l='MpServer', x=-71,50, y=16,00, z=-414,50], EntityTrail['unknown'/186, l='MpServer', x=-66,54, y=75,62, z=-457,13], EntitySkeleton['Skeleton'/700, l='MpServer', x=15,54, y=-34,62, z=23,13]]
	Retry entities: 0 total; []
	Server brand: fml,forge
	Server type: Integrated singleplayer server
Stacktrace:
	at net.minecraft.client.multiplayer.WorldClient.func_72914_a(WorldClient.java:373)
	at net.minecraft.client.Minecraft.func_71396_d(Minecraft.java:2444)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:927)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows 8.1 (amd64) version 6.3
	Java Version: 1.8.0_31, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 2624048768 bytes (2502 MB) / 3207856128 bytes (3059 MB) up to 3207856128 bytes (3059 MB)
	JVM Flags: 7 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx3G -Xms3G -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 12, tallocated: 95
	FML: Minecraft 1.7.10 MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.4.1614 Optifine OptiFine_1.7.10_HD_U_D3 37 mods loaded, 37 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAAAA	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAA	FML{7.10.99.99} [Forge Mod Loader] (forge-1.7.10-10.13.4.1614-1.7.10.jar) 
	UCHIJAAAA	Forge{10.13.4.1614} [Minecraft Forge] (forge-1.7.10-10.13.4.1614-1.7.10.jar) 
	UCHIJAAAA	CodeChickenCore{1.0.4.35} [CodeChicken Core] (minecraft.jar) 
	UCHIJAAAA	NotEnoughItems{1.0.4.95} [Not Enough Items] (NotEnoughItems-1.7.10-1.0.4.95-universal.jar) 
	UCHIJAAAA	OpenEye{0.6} [OpenEye] (OpenEye-0.6-1.7.10.jar) 
	UCHIJAAAA	mod_ThreadedLighting{1.7.10-1.0} [Threaded Lighting] (minecraft.jar) 
	UCHIJAAAA	gilded-games-util{1.7.10-1.1} [Gilded Games Utility] (gilded-games-util.jar) 
	UCHIJAAAA	battlegear2{1.7.10} [Mine & Blade Battlegear 2 - Bullseye] (1.7.10-MB_Battlegear2-Bullseye-1.0.8.3.jar) 
	UCHIJAAAA	lucky{5.1.0} [Lucky Block] ([1-7-10]_Lucky_Block_v5-1-0.jar) 
	UCHIJAAAA	adventurebackpack{1.7.10-0.8b} [Adventure Backpack] (adventurebackpack-1.7.10-0.8c.jar) 
	UCHIJAAAA	aether{Alpha 1.7.10-1.2} [Aether II] (aether.jar) 
	UCHIJAAAA	AnimationAPI{1.2.4} [AnimationAPI] (AnimationAPI-1.7.10.jar) 
	UCHIJAAAA	atum{0.6} [Atum] (Atum-1.7.10-0.6.77.jar) 
	UCHIJAAAA	BiomesOPlenty{2.1.0} [Biomes O' Plenty] (Biomes-O-Plenty-Mod-1.7.10.jar) 
	UCHIJAAAA	MetallurgyCore{4.0.4} [Metallurgy Core] (MetallurgyCore-1.7.10-4.0.4.18.jar) 
	UCHIJAAAA	Metallurgy{4.0.6} [Metallurgy 4] (Metallurgy-1.7.10-4.0.6.80.jar) 
	UCHIJAAAA	B0bGrowsOre{2.5.4} [B0bGary's Growable Ores] (B0bGary's Growable Ores-2.5.4 for 1.7.10.jar) 
	UCHIJAAAA	BetterAchievements{0.1.0} [Better Achievements] (Better-Achievements-Mod-1.7.10.jar) 
	UCHIJAAAA	CarpentersBlocks{3.3.7} [Carpenter's Blocks] (Carpenter's Blocks v3.3.7 - MC 1.7.10.jar) 
	UCHIJAAAA	chocolateQuest{1.0} [Chocolate Quest] (chocolateQuest-1.7.10-1.0.jar) 
	UCHIJAAAA	DamageIndicatorsMod{3.2.0} [Damage Indicators] (Damage-Indicators-Mod-1.7.10.jar) 
	UCHIJAAAA	DragonMounts{r41-1.7.10} [Dragon Mounts] (Dragon-Mounts-Mod-1.7.10.jar) 
	UCHIJAAAA	gravestonemod{1.7.10} [Gravestones] (GraveStones 1.7.10 -3.jar) 
	UCHIJAAAA	iChunUtil{4.1.3} [iChunUtil] (iChun-Util-Mod-1.7.10.jar) 
	UCHIJAAAA	InfernalMobs{1.6.6} [Infernal Mobs] (Infernal-Mobs-Mod-1.7.10.jar) 
	UCHIJAAAA	MobDismemberment{4.0.0} [MobDismemberment] (Mob-Dismemberment-Mod-1.7.10.jar) 
	UCHIJAAAA	Morph{0.9.2} [Morph] (Morphing-Mod-1.7.10.jar) 
	UCHIJAAAA	MutantCreatures{1.4.9} [Mutant Creatures] (Mutant-Creatures-Mod-1.7.10.jar) 
	UCHIJAAAA	primitivemobs{1.0} [Primitive Mobs] (primitivemobs-1.0b-1.7.10.jar.zip) 
	UCHIJAAAA	Rediscovered{1.1} [Minecraft Rediscovered Mod] (Rediscovered-Mod-1.7.10.jar) 
	UCHIJAAAA	ReiMinimap{1.7.10} [Rei's Minimap] (Reis-Minimap-Mod-1.7.10.jar) 
	UCHIJAAAA	bspkrsCore{6.15} [bspkrsCore] ([1.7.10]bspkrsCore-universal-6.15.jar) 
	UCHIJAAAA	Treecapitator{1.7.10} [Treecapitator] (Tree-Capitator-Mod-1.7.10.jar) 
	UCHIJAAAA	TwilightForest{2.3.7} [The Twilight Forest] (twilightforest-1.7.10-2.3.7.jar) 
	UCHIJAAAA	Waila{1.5.10} [Waila] (Waila-Mod-1.7.10.jar) 
	UCHIJAAAA	weaponmod{v1.14.3} [Balkon's WeaponMod] (weaponmod-1.14.3.jar) 
	GL info: ' Vendor: 'NVIDIA Corporation' Version: '4.5.0 NVIDIA 368.22' Renderer: 'GeForce GTX 770/PCIe/SSE2'
	Launched Version: 1.7.10-Forge10.13.4.1614-1.7.10
	XRay Version: 15
	LWJGL: 2.9.1
	OpenGL: GeForce GTX 770/PCIe/SSE2 GL version 4.5.0 NVIDIA 368.22, NVIDIA Corporation
	GL Caps: Using GL 1.3 multitexturing.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Anisotropic filtering is supported and maximum anisotropy is 16.
Shaders are available because OpenGL 2.1 is supported.

	Is Modded: Definitely; Client brand changed to 'fml,forge'
	Type: Client (map_client.txt)
	Resource Packs: []
	Current Language: English (US)
	Profiler Position: N/A (disabled)
	Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	Anisotropic Filtering: On (8)
	OptiFine Version: OptiFine_1.7.10_HD_U_D3
	Render Distance Chunks: 18
	Mipmaps: 4
	Anisotropic Filtering: 8
	Antialiasing: 0
	Multitexture: false
	OpenGlVersion: 4.5.0 NVIDIA 368.22
	OpenGlRenderer: GeForce GTX 770/PCIe/SSE2
	OpenGlVendor: NVIDIA Corporation
	CpuCount: 8
[17:06:06] [Client thread/INFO] [STDOUT]: [net.minecraft.client.Minecraft:func_71377_b:359]: #@!@# Game crashed! Crash report saved to: #@!@# C:\Users\Dario\AppData\Roaming\.minecraft\crash-reports\crash-2016-05-27_17.06.05-client.txt
```

**Comment by iChun** [comment]
This issue is being closed as a purge of issues due to the Morph rewrite to 1.12.2 finally being released. Support for the 1.7.10 version has long been dropped and focus has shifted to the 1.12.2 version of the mod instead.
